### ISSUE TITLE
[discussion] type definition of `jobs` attribute of `Bree.BreeOptions` seems incorrect (or not working as expected)

### ISSUE THREAD

**Issue by ksze** [breejs/bree#165]
#### What would you like to discuss?
 
In the examples, the `jobs` array can take objects that are simply like this:

```typescript
{
  name: 'my_awesome_job',
}
```

And Bree would look for the `my_awesome_job.js` or `my_awesome_job.ts` file under the jobs root folder.

However, when I try to use that convention, TypeScript tells me that the `path`, `interval`, and `timeout` are also required, contrary to the examples.

I'm using TypeScript 4.6.2 and Bree 8.0.1

Here is a screenshot from VS Code:
![image](https://user-images.githubusercontent.com/381556/158506420-9cc33dcd-8a23-449d-8303-5f5813c51a9c.png)

#### Checklist

- [x] I have read the documentation.

**Comment by ksze** [comment]
I suspect that this needs to be changed:
```typescript
type BreeOptions = Partial<BreeConfigs> & {
  jobs?: Array<string | (() => void) | JobOptions>;
};
```
It needs to become this:
```typescript
type BreeOptions = Partial<Omit<BreeConfigs, "jobs">> & {
  jobs?: Array<string | (() => void) | JobObtions>;
};
```

**Comment by shadowgate15** [comment]
Fixed! Update to v8.0.2.
